### DISCUSSION TITLE
Can you please add the coloured brackets feature of this plugin for C# to the free version of Rainbow Brackets?

### DISCUSSION THREAD

**Discussion by kris3713** [izhangzhihao/intellij-rainbow-brackets#2693] [Polls]
### Have you checked the issues and discussions to ensure there are no duplicates?

Yes

### Your programming languages

C#

### Free or paid?

Free

### Expected Behavior

I'm not asking for much, can you please at least provide just the coloured brackets feature for C# in the free version of the plugin? I don't care if I am loosing other features like rainbow indentation, synaptic highlighting, or different coloured variables, I just want the basic coloured brackets feature.

### Code snippet for test

```csharp
using System;
using System.Windows.Forms;

namespace TestExample {
    internal static class Program {
        /// <summary>The main entry point for the application.</summary>
        [STAThread]
        private static void Main() {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            Application.Run(new Form1());
        }
    }
}
```

### Your Environment

JetBrains Rider 2023.2.3
Build #RD-232.10203.29, built on November 1, 2023
Licensed to Zliced13
Runtime version: 17.0.8.1+7-b1000.32 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
.NET Core v7.0.7 x64 (Server GC)
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry:
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    ide.new.project.model.index.case.sensitivity=true
    vcs.empty.toolwindow.show=false
    http.client.file.variables.available=true
    database.show.search.tab=false

Non-Bundled Plugins:
    me.laria.code.idea_caseconv (0.7.1)
    eu.osimowicz.plugins.intellij.CodeBlocksSorter (0.1.2)
    lermitage.intellij.extratci (1.27.1)
    com.intellij.resharper.HeapAllocationsViewer (2023.2.0)
    Identifier Highlighter Reloaded (1.02.1)
    MultiHighlight (3.1.0)
    com.nasller.CodeGlancePro (1.7.6)
    com.github.catppuccin.jetbrains (3.2.1)
    com.jeno.bracket-selection (1.7.3)
    com.intellij.ideolog (222.1.0.0)
    com.sburlyaev.terminal.plugin (0.6.0)
    nl.bryanderidder.regexrenamefiles (1.2.5)
    com.codeium.intellij (1.4.11)
    fr.socolin.application-insights-debug-log-viewer (1.9.0)
    com.furstenheim.more-actions (1.7-SNAPSHOT)
    eu.gillissen.rider.usersecrets (1.2.0)
    Key Promoter X (2023.3.0)
    hunspell (232.10203.26)
    indent-rainbow.indent-rainbow (2.2.0)
    com.pinkfloyded.idea-file-path-autocomplete.idea-file-path-autocomplete (0.4.1)
    de.achimonline.github_markdown_emojis (1.3.2)
    com.intellij.grazie.pro (0.3.265)
    tanvd.grazi (232.9921.48)
    HighlightBracketPair (1.4.2)
    com.github.blackhole1.ideaspellcheck (0.0.7)
    net.seesharpsoft.intellij.plugins.csv (3.2.3-232)
    uk.co.ben-gibson.remote.repository.mapper (4.3.6)
    com.mallowigi (93.1.0)
    mobi.hsz.idea.gitignore (4.5.2)
    cn.yiiguxing.plugin.translate (3.5.6)
    izhangzhihao.rainbow.brackets.lite (1.1.0)

**Comment by izhangzhihao** [comment]
I will convert this to a vote.

**Reply by kris3713** [reply]
hey did you forget?
